docs/windows.md

```diff
@@ -103,6 +103,20 @@ higher (containerd 1.7 or higher is recommended). It relies on support for
 [Windows HostProcess Pods](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/),
 which is generally available starting with K8s 1.26.
 
+More detailed containerd Version requirements are outlined below:
+
+| Kubernetes Version  | Recommended containerd Version |
+| ------------------- | ---------------- |
+| 1.26                | 1.7.0+, 1.6.18+  |
+| 1.27                | 1.7.0+, 1.6.18+  |
+| 1.28                | 1.7.0+, 1.6.18+  |
+| 1.29                | 1.7.11+, 1.6.27+ |
+| 1.30                | 1.7.13+, 1.6.28+ |
+
+Note: Starting from Antrea v2.1, Antrea Windows image is built based on the hpc image, containerd version 1.6.18 or higher is required because versions earlier than 1.6.18 do not support importing HPC images on Windows.
```

Comment:
```suggestion
Note: Starting from Antrea v2.1, Antrea Windows image is built based on the HPC (Host Process Containers) image, 
containerd version 1.6.18 or higher is required because versions earlier than 1.6.18 do not support
importing HPC images on Windows.
```
Please make the line length around 80-100 chars.

Reply:
Updated, thanks.